src/libUtils/GetTxnFromFile.h

```diff
@@ -29,21 +29,27 @@
 
 #include "Logger.h"
 #include "libData/AccountData/Transaction.h"
-
-unsigned int TXN_SIZE = Transaction::GetMinSerializedSize();
+#include "libMessage/Messenger.h"
 
 bool getTransactionsFromFile(std::fstream& f, unsigned int startNum,
```

Comment:
This function should get the txns starting with nonce startNum, not from the file beginning

Reply:
Now i read all transactions from the file, and then filter out the transactions needed. Because with protobuf, it cannot read from the middle of the data.